### ISSUE TITLE
Symbol in the module exports array causes fatal error

### ISSUE THREAD

**Issue by andrew-yustyk** [nestjs/nest#1246]
## I'm submitting a...
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Currently it isn't possible to use symbol in the exports array. 
An example:
```typescript
// sentry.constants.ts
export const SENTRY_SDK_TOKEN = Symbol('SENTRY_SDK_TOKEN');
```
```typescript
// sentry.module.ts
@Module({
  imports: [ConfigModule],
  providers: [
    {
      provide: SENTRY_SDK_TOKEN,
      useFactory: SentryProvider.fromConfigService,
      inject: [ConfigService]
    }
  ],
  exports: [SENTRY_SDK_TOKEN]
})
export class SentryModule {
}
```
The bug is not reproduced if you use the string instead of the symbol.

Seems like this is a bug in the `addExportedComponent()` method in the injector module [(link).](https://github.com/nestjs/nest/blob/master/packages/core/injector/module.ts#L289)

Current implementation of this method doesn't have a checking of the symbol token, just for the string token.

## Expected behavior
Symbol token can be used in the exports section as well as the string token

## Environment

<pre><code>
Nest version: 5.4.0

For Tooling issues:
- Node version: 10.11.0
- Platform:  Linux
</code></pre>

**Comment by andrew-yustyk** [comment]
There is just a bug or missed functionality. I'll try to create PR for for this issue asap

**Comment by marcus-sa** [comment]
Oh, my bad then

**Comment by andrew-yustyk** [comment]
there is need just to update `addExportedComponent()` method in in the injector module

**Comment by marcus-sa** [comment]
Anyhow, you should use `Symbol.for` and not `Symbol`

**Comment by andrew-yustyk** [comment]
I think this is depends on your project.
I prefer to use `Symbol('some_str')` instead of `Symbol.for('some_str')`. 
`Symbol.for` allows other developers do not import the constant with symbol and just use `Symbol.for` in different places of project. And then an additional time will be spent during refactoring.

In a case of `Symbol` other developers must import the constant with this symbol.

**Comment by marcus-sa** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol

There's a big difference.

**Comment by kamilmysliwiec** [comment]
Thanks for reporting!

**Comment by kamilmysliwiec** [comment]
Published as 5.4.1
